### PR TITLE
[do not merge] Add yast-users tests

### PR DESCRIPTION
**Pull request by imobach**
Add some basic tests about yast2-users (create, edit and delete users). It contains a test about fixed bug [bsc#928607](https://bugzilla.suse.com/show_bug.cgi?id=928607) so I guess it should not be merged until yast2-users 3.1.39 is available on TW.

Anyway, feedback is welcome.

Thanks.

### CODE REVIEW

**Comment by coolo** [comment]
:+1:

File: tests/console/yast2_users.pm [hunk 1]
```diff
@@ -0,0 +1,84 @@
+use base "console_yasttest";
+use testapi;
+
+sub add_user() {
+    send_key "alt-a";
+    assert_screen "yast2_users-add_user";
+    type_string "Test user";
+    assert_screen "yast2_users-username";
+    send_key "alt-p";
+    type_string "nots3cr3t.";
```
Comment (lnussel) [inline tests/console/yast2_users.pm]: type_password()

File: tests/console/yast2_users.pm [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    send_key "alt-t";
+    assert_screen "yast2_users-confirm_delete";
+    send_key "alt-y";
+    assert_screen "yast2_users-users";
+}
+
+sub add_passwordless_user() {
+    script_run "pam-config -a --unix-nullok";
+    script_run "useradd test -c 'Test user'";
+    script_run "passwd -d test";
```
Comment (lnussel) [inline tests/console/yast2_users.pm]: you may want to use the new assert_script_run

**Comment by imobach** [comment]
@lnussel Thanks for your feedback. BTW, `assert_script_run` is really handy :) I've updated the PR.

File: tests/console/yast2_users.pm [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+sub add_user() {
+    send_key "alt-a";
+    assert_screen "yast2_users-add_user";
+    type_string "Test user";
+    assert_screen "yast2_users-username";
+    send_key "alt-p";
+    type_password "nots3cr3t.";
+    send_key "tab";
+    type_password "nots3cr3t.";
```
Comment (nilxam) [inline tests/console/yast2_users.pm]: type_password; please :)

Comment (imobach) [inline tests/console/yast2_users.pm]: Done. Thanks!

**Comment by sysrich** [comment]
Is there a matching SLE12 submission to activate this test there?

**Comment by coolo** [comment]
close or merge?

**Comment by imobach** [comment]
@coolo Hi, as it looks like the way to test YaST modules on OpenQA is changing, I'd prefer closing and I'll submit them if needed.

**Pull request closed without merging**